src/Umbraco.Web.UI.NetCore/Umbraco.Web.UI.NetCore.csproj

```diff
@@ -4,35 +4,52 @@
         <TargetFramework>netcoreapp3.1</TargetFramework>
         <RootNamespace>Umbraco.Web.UI.NetCore</RootNamespace>
     </PropertyGroup>
-
     <ItemGroup>
+      <ProjectReference Include="..\Umbraco.Examine.Lucene\Umbraco.Examine.Lucene.csproj" />
```

Comment:
We need a task to change this to work properly since this in theory shouldn't actually be working :) We won't be able to ship with Lucene 3.0.3, we'll need to update Examine to Lucene 4.8.0 (netstandard) which is currently in beta00008 (though it is considered stable)

Reply:
Isn't this the task? you Created it yourself?
https://dev.azure.com/umbraco/D-Team%20Tracker/_workitems/edit/4911

Reply:
haha, well i guess so :)